Added note about PKCE usage with OIDC plugin (#5350)
Kong/docs.konghq.com: app/_hub/kong-inc/openid-connect/_index.md

<img src="/assets/images/docs/openid-connect/authorization-code-flow.svg">

> {:.info}
> > If PKCE is to be used, the Identity Provider _must_ contain the ```code_challenge_methods_supported``` object in the ```/.well-known/openid-configuration``` Issuer discovery endpoint response. If it is not included, the PKCE ```code_challenge``` query parameter will not be sent. The ```code_challenge_methods_supported``` is a requirement for PKCE per [RFC 8414](https://www.rfc-editor.org/rfc/rfc8414.html).

Reviewer: ```suggestion
{:.note}
> If using PKCE, the identity provider *must* contain the `code_challenge_methods_supported` object in the `/.well-known/openid-configuration` issuer discovery endpoint response, as required by [RFC 8414](https://www.rfc-editor.org/rfc/rfc8414.html). If it is not included, the PKCE `code_challenge` query parameter will not be sent.
```
